### ISSUE TITLE
workaround for ruby 2.7 deprecation warnings

### ISSUE THREAD

**Issue by draptik** [athityakumar/colorls#330]
### Description

After a system update which installed ruby version 2.7 the following deprecation warnings are displayed when calling `colorls`:

```sh
$ colorls
/usr/lib/ruby/gems/2.7.0/gems/colorls-1.2.0/lib/colorls/flags.rb:50: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/usr/lib/ruby/gems/2.7.0/gems/colorls-1.2.0/lib/colorls/core.rb:5: warning: The called method `initialize' is defined here
```
Prepending `RUBYOPT=-W:no-deprecated` to the `colorls` command will remove these warnings. Example usage:

```sh
RUBYOPT=-W:no-deprecated colorls
```

```sh
$ ruby --version
ruby 2.7.0p0 (2019-12-25 revision 647ee6f091) [x86_64-linux]
```

**Comment by avdv** [comment]
Thanks! I've created PR #331 which fixes the warning.

**Comment by avdv** [comment]
You may use the pre-release (once it's build and pushed to ruby gems) if you want.

**Comment by draptik** [comment]
Thx for the quick fix! :clap: 

And the heads-up on using the pre-release (although i'll wait for the arch-maintainer to update to the upcoming release)!

**Comment by ykonstant1** [comment]
Unfortunately in my case with ruby 2.7 I get errors of the form
```
Traceback (most recent call last):
	2: from /usr/bin/colorls:23:in `<main>'
	1: from /usr/lib/ruby/2.7.0/rubygems.rb:294:in `activate_bin_path'
/usr/lib/ruby/2.7.0/rubygems.rb:275:in `find_spec_for_exe': can't find gem colorls (>= 0.a) with executable colorls (Gem::GemNotFoundException)
```
and I had to downgrade the ruby ecosystem to get colorls back running.

**Comment by avdv** [comment]
@ykonstant1 did you reinstall colorls after upgrading Ruby? It would be good if you can tell us how to reproduce...

**Comment by ykonstant1** [comment]
@avdv I see; after updating ruby, I did `gem update colorls`, and it reported nothing to update.  Prompted by your comment I did `gem install colorls` instead, and now it is working ok.

**Comment by avdv** [comment]
I just released version 1.3.0 of colorls, so I consider this fixed. (I just wanted to keep it open so people can find it easily when they stumble upon the same issue)
